server/services/lobby_service.go

```diff
@@ -74,6 +82,25 @@ func (l *LobbyServiceImpl) JoinLobby(lobbyID string, playerID string) (Lobby, er
 	}
 
 	lobby.Players = append(lobby.Players, playerID)
+	lobby.PlayerReady[playerID] = NotReady
+	return *lobby, nil
+}
+
+// ReadyPlayer will toggle the player's ready status in the lobby, if they exist, or an error
+func (l *LobbyServiceImpl) ReadyPlayer(lobbyID string, playerID string) (Lobby, error) {
```

Comment:
I might argue to have this be more explicit in what our desired 'readiness' is: aka pass in a boolean of desired ready status.

Reply:
To make this work nicely, it may make sense to make a `PlayerReadyMsg` that had the player ID/Name and the desired readiness state (that way a player can mark themselves as not ready, if they need to).

We can then send that message as the body of the request to the `/ready` endpoint.